src/helpers/ApiHelper.js

```diff
@@ -249,6 +256,7 @@ const ApiHelper = {
     });
   },
   postCheckOutItem (apiUrl, token, errorHandlerFn, item, callback) {
+    logger.info('checking out');
```

Comment:
should this indicate what it's checking out, or are the batches of items small enough that it's clear in context?

Reply:
Just one at a time, but I think it's probably a good idea to add some indication of which item is being checked out. I'll add that in.